src/tests/unit-tests/system-parameters/beidou_cnav2_nav_message_test.cc

```diff
@@ -0,0 +1,661 @@
+/*!
+ * \file beidou_cnav2_navigation_message_test.cc
```

Comment:
You need to add the test to the main test file located at `gnss-sdr/src/tests/test_main.cc`. Please refer to the template I offered a while ago for those details